utils/imgutils/gif.go

```diff
@@ -501,10 +481,34 @@ func (d *decoder) readBlock() (int, error) {
 	return int(n), nil
 }
 
-func CountFrames(r io.Reader) (int, error) {
+func CountGIFFrames(r io.Reader) (int, error) {
```

Comment:
Nit: is there any case of frames outside of gifs? i.e. do we need to change the function name?

Reply:
Well, part of the reason I took a while to figure out what was going on is that the name was very anonymous. I don't think it hurts to be more explicit and this method is clearly for internal use only. To better answer your question, there are other formats that support animation like WebP or AVIF (likely the next GIF).

Reply:
Got it, makes sense :+1: